test/unit/content-info.test.js

```diff
@@ -23,6 +23,23 @@ describe('contentInfo module', function () {
RphOdD5iaacEM1pyoEHgvq65Z1wftr/xAW+wSmfD9N4sGlsQQYXCYWHATaE0AyktOEj6hccRQlM2bSZ6EDEXwnMfnEMmje1w64+lIEgHRHqmJa1BE0G4ALcY2ECpkJojDRhbOfPBxEIggPTcihGYpQSYeqa1BTc/614M5ZDpRjInRFomf/DyrGl6lNGd5COLCevsUfkPZI9f+5yU8m//uSZKUAAo4Zz+MMWxxSIzlKY0s3CjxVOSyxbLFQjORNjbyYhAO52RIlAtQ7SN1PdwVyYhJWYskyLI8W2pdIzLZXQBHsDb+e1HgYGkMYGBYRtOyM4bN6LT67GvSX9CKl9K0vb93/+2j+1elr20+29XC7QQ6CxrcQA9LOJEoQIgZKKyPLorSZa7Y9GmVTR8Vvqx7UlmAPr5ql3hwmZnpy4mK0RqQglFp6WCTwjeUYFlOCHyCO+w9YjJr+9dmtDyZTNB0iokUDQZFXpcAAaCycaTACLQeCmTxsBXAW6ly0SIi+I0A/oMyHnVL3EcsCLRL21RC7Ao7913kRhxFIbjw0qoUt3ODO1LiKRcgIxoWNHDYYUpH/fXpftwmkA5+hAE5FY2kmAKGTluU76rCs7AxALlwwa/KJIeFMcRIg5axOtkiFzhuJNvEA8nbtPg4L0JhkUm5snytsgq+GTbzx2o0GJ5r+n7H/qD2r6PzyBiGBxLCFSCdZRtEAhUCfINDcArEghgCmLD4pATd3qFIYdWo6MRCzTNdh63E3bePoJsVGIDrTdv/7kmS/AAKXSlFR6BReU4OZ6mGCa4pMVy9MsW5BT4zmKaUyEmjEtN4exljbilipqVtjBKOCw8tfIjUMU3Scipp1nYysQ76RQWADwIfWdEKQAABQAcqgGkAbIyhOAy4FxA1AEAsensDGU3UtL3hd0MtRzBMYuEGBwyBBUQa43SGn4qwrewEE9z+0+FQG2TBE0Mo8HDaTPqZa4ynV/67tQk9f//XCCVbjaRQAKEUCeUmNXSYcoyC30TlruPsp5ckWF744AK2D5wy6ph/NnzH7gnS1ZpEvrkxLmhHuRFMjivS04HPhd4qzKQp/9zO/n5rPMuV9z/9NFe25qDwcwUAOAE26RFUENGREYaaQRpQAmUzeIRKcLA5iECqAEZjDmeHLkHAt1W/Byqgoz/Y7BwUlIxNt+5/DyMZiXne6L+4w3NAaPKEq1qpVjnedXZ9n///2qgWWAxwAMAITJ0U/4LMeujXfU18lM7HUiRJ3R9HCYxATMhViIEDgKEw8PKOq0YnKNru9AqMxS//2cp/bOdvFWxRZ71HXyWosVFIE0c8jdvtkmhL/+5Bk14CC5CRPUy1DrFOi2Qxjaw4K9RNBTLBteVGLI8mePGihwov2FlP/5M8+coJlclPx9X+HiaSA8YmDjRhKsbqvmtLRt+OYQxmcjBjCIeCIOMZK8m6ohJrl2V8OPRDx0+zIH2exSDxdEjzGXK1TUOxnE3pWMpY16t5OPJZBa7xmGRaNYojrbpUTOhIBDrn7BBAIOGjhZoJWawhH3Hgo9GO+ImuGnz5r4KceDmRIhlBodqUmZtIYUoWOahzAwvLn5j6t6YbUrygBUGBQXRoGJFjGfcS4GtkQmKp9PzOnfaqWTnmbFfzOf/uvVSHM+lOPllTKT9+Pn/8xfreMniycqEcckIIh81WAAC1SIsoI5mCGBBYXRioARx0jjUlRCmMqFS7EQ07gQoSCwsN2MrC4OE+Xjx5fLh6Niuo2QXMZATmYJD3b/jPOAYmrDkKNp6tKvmVmwREg0+TGhe4vuuvXnS8OSAEFAubqhUWDDUIDAkSTCsrhpcDA8pDCAOR5pTYrKTUESTCscw4HzC8kTuvDBEQEINARgZpAJEOrMwcSAXj/+5Jk54wDKT3Ik2kcoltEGPJt44YOsUMiTaRySYGM5emnraSU9qZjmfIgQxFAC8FBbwgk8y3dCJcrxNnKaZegjqz7Om35aTxJT+UjLOD31NcWRK7//7rDG5cwZQzZkd/WiEibbZrAxAjgtBPwVywD5eCaDcI0eo5BmmdZY1yY0cVYnivXeXTpxCFS1xAM3jzK0ecPBxbKvIX//ikZb//+KLSWcxC4wA2m0k4FPiAATAYShYlQ4Ah4TDK4WjNyTjoUABEpgQaVD07YoMFBC6xMryKkQNil3rdUYXftSxl1FWnhmtyC02mSk4+NSRW98/LSrOjyGCIrE/lV+4yc11Op92/wc8Rta8loQnjFtF9dXhmymwuvEA6wnJsgNPTW3ANwQAzIdOTirL5fhjzyRRTAqMB13HgVXfMiLUQd6mjG6a9jDsjRZ7+l4N2w9m1MPKiodNvaV//8h70sd0sZ/IIYdZGqXekAOUqwAjCYgApiljA5GmGwwYoQxtQJHj5Af4YGyHghNjGCA/YbCpeAgMyoQa7H0478Vnn0Jg1XGdhRvPdf//uSZNsIA6Q5yJO6GsJGAwqdPMeRjlh/KU7sx4kvCuo1hioWDvJXTaHWMCafTDBRsF4QW1nLXS/TJfzyU4OUa716qrkVznO1aVv2/53x+2r6fj657RW7uGXr60QYZHbLWwGFSIThR48esylWEKdTDV7DZreU2WzXJBXYAT4xq1DqJD4JO0qytNXQWGTHOQtHlUBQmPBscDo0Ddn0jOa////oe1h4HElyBjVRYMKgLQDAwsBEAhe8wyMYwIMEzc38wpGMwQA8DCeSEAfvpnZAKoZhlUofq87UNp2OCknudZRnvcWh2Zr+9EH3rH2t72gJyHKamhaQ4MZw7s0zopXuZHX+zkru+6IzX50QWm4b18A/qsrflt9x9X8K/+U0HzQKTtuusDECCQrOVdqTj7OGgiACv3ZjpKUTwRx8P3UpGdMLakTgQ7dprdQXyaWFA1O8lLX6TpMah5LSCwZHNtyH/oqyrt3/65RZRcAHg0VXlRWDAcHiIOwYdw8MRhKG5g8UhkSK5rlExmeDIFDcwUDUwdA0eg4wjFYeCQ40p5REZRCIYQ==',
```

Comment:
Let's move this to `test/fixtures/audio.mp3`

Reply:
done